components/Favourites/Favourites.js

```diff
@@ -0,0 +1,132 @@
+import styled from "styled-components";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import PropTypes from "prop-types";
+
+import { Divider, ItemCard } from "../../styles/UIKit";
+import { FAVOURITES_TITLE } from "../../utils/constants";
+
+const responsive = {
+  superLargeDesktop: {
+    // the naming can be any, depends on you.
+    breakpoint: { max: 4000, min: 3000 },
+    items: 5,
+  },
+  desktop: {
+    breakpoint: { max: 3000, min: 1024 },
+    items: 5.5,
+  },
+  tablet: {
+    breakpoint: { max: 1024, min: 464 },
+    items: 2,
+  },
+  mobile: {
+    breakpoint: { max: 464, min: 0 },
+    items: 1,
+  },
+};
+
+let nextOne, previousOne;
+
+const ButtonGroup = ({ next, previous, ..._ }) => {
+  nextOne = next;
+  previousOne = previous;
+  return <div></div>;
+};
+
+ButtonGroup.propTypes = {
+  next: PropTypes.func,
+  previous: PropTypes.func,
+};
+const GotoNext = () => {
```

Comment:
These functions should be inside the component, and names should have camel casing.